dev/http.go

```diff
@@ -178,6 +179,23 @@ func (h *HTTPServer) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 	}
 }
 
+func (h *HTTPServer) shouldServePublicPathForApp(a *App, req *http.Request) bool {
+	reqPath := path.Clean(req.URL.Path)
+
+	if !a.Public {
+		return false
+	}
+
+	if reqPath == "/" {
+		return false
+	}
+
+	// FIXME: return false if any of h.IgnoredStaticPaths "contains" reqPath
+	fmt.Printf("reqPath:%s ignoredPaths:%s\n", reqPath, h.IgnoredStaticPaths)
```

Comment:
@Pezmc i threw together this impl a couple weeks ago, but it's wrong, as you might guess from the failing tests. 🙃  I think the tests are there, but if you can provide a working implementation of this function, I think that'd do it. I don't have time to dig in today, but should be able to take a look by EOW.